upf/upf_pfcp_api.c

```diff
@@ -524,7 +524,7 @@ handle_association_setup_request (pfcp_msg_t * req,
        * PFCP Association Setup Response message.
        *
        */
-      pfcp_release_association (n);
+      upf_pfcp_server_deferred_release_association (n - gtm->nodes);
```

Comment:
Rel. 16 changed the rules on this.

**just a heads up, don't change it now!**

Reply:
✓
Thanks